frontend/packages/operator-lifecycle-manager/src/components/k8s-resource.tsx

```diff
@@ -120,9 +121,9 @@ export const ResourceTable: React.FC<ResourceTableProps> = (props) => {
   );
 };
 
-export const flattenCsvResources = (parentObj: K8sResourceKind) => (resources: {
-  [kind: string]: { data: K8sResourceKind[] };
-}): K8sResourceKind[] => {
+export const flattenCsvResources = (
+  parentObj: K8sResourceKind,
+): Flatten<{ [key: string]: K8sResourceKind[] }, K8sResourceKind[]> => (resources) => {
```

Comment:
Can we use `K8sResourceCommon` here or does it cause a problem?

Reply:
works fine with `K8sResourceCommon` - fixed